app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -34,6 +34,12 @@ def failure
   private
 
   def callback_for(provider)
+    # Deleting the session cookie with the legacy app domain, which does NOT include a preceding dot.
+    # This should fix the double cookie scenario.
+    # TODO: this code is a hotfix, we should remove it after 09/18/2020.
+    legacy_cookie_domain = Rails.env.production? ? ApplicationConfig["APP_DOMAIN"] : nil
```

Comment:
Since the cookie we want to delete does _not_ include the preceding dot, we can just pull it directly from `ApplicationConfig["APP_DOMAIN"]` and delete it.